### PR TITLE
Express async errors

### PR DESCRIPTION
**Pull request by lukasaric**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const helmet = require('helmet');
 const origin = require('./shared/origin');
 const passport = require('passport');
 const path = require('path');
+require('express-async-errors');
```
Comment (abasic) [inline server/app.js]: @vladimyr On line 14. there is a require that is assigned to an unused variable. Is there a reason why we would want to do this in different ways at different times or should we consolidate it?

Comment (vladimyr) [reply server/app.js]: I'm all for removing it and just requiring stuff at appropriate place without that eslint pragma above.

**Comment by vladimyr** [comment]
@lukasaric Please resolve reported conflicts :wink:

File: server/app.js (hunk 1 above)
Comment (lukasaric) [reply server/app.js]: > @lukasaric Please resolve reported conflicts 😉

@vladimyr unused auth variable problem with unnecessary eslint pragmas fixed :point_down: .
Question, should we refactor auth/passport setup like in boutique, or should we leaving it as it is?

Comment (vladimyr) [reply server/app.js]: If that means new pr go for it 😁

Comment (lukasaric) [reply server/app.js]: :+1:

**Comment by vladimyr** [comment]
Recreated PR due to snyk test failure 🤦‍♂ 
@abasic Leaving to you to hit the green button 😉

**Pull request merged**